### PR TITLE
[Feature] Updated Gradle and added Github Action 

### PR DESCRIPTION
**Pull request by ttpho**
- Updated Gradle version 
- Updated Libs version
- Fixed bug
`NoClassDefFoundError: Failed resolution of: Ljava/awt/Color `
- Added Github Action:

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -21,6 +21,15 @@ android {
                 }
             }
         }
+        debug {
```
Comment (ttpho) [inline app/build.gradle:24]: @depapp I set APK file name is `app-debug.apk` for GitHub Action

Comment (depapp) [reply app/build.gradle:24]: great @ttpho , thanks 🍻

**Comment by ttpho** [comment]
@depapp Github Action builds success. Thank you for your contact 
https://github.com/ttpho/android-inspector/actions/runs/3224820208

**Comment by depapp** [comment]
> @depapp Github Action builds success. Thank you for your contact https://github.com/ttpho/android-inspector/actions/runs/3224820208

great. thanks @ttpho

**Pull request merged**